README.md

```diff
@@ -1,4 +1,5 @@
 # BEP_1920_Q2
+[![BCH compliance](https://bettercodehub.com/edge/badge/IssaHanou/BEP_1920_Q2?branch=develop-sprint-7)](https://bettercodehub.com/)
```

Comment:
probably more informative description is better here

Reply:
its a badge, render markdown